docsearch/algolia_configs/latest_commit.json

```diff
@@ -22,6 +23,7 @@
     "en-tidb-v3.0": "ff3bdb392d243f7a48051e037fb750579e7a6ca5",
     "en-tidb-v2.1": "2755565832044b9a1fdc4be01057dc398098e6f3",
     "zh-tidb-dev": "f27485b695f5955aac3922b4b5e908d0950ca854",
+    "zh-tidb-v7.5": "01c68823bbd67a05b9002e8bc8754b3b24341292",
```

Comment:
The latest commit SHA of the [release-7.5](https://github.com/pingcap/docs-cn/commits/release-7.5) branch of [docs-cn](https://github.com/pingcap/docs-cn) is `d6845f592a72823f5c1d4f96bb9ed456e316c42f`. Do we need to use the latest?